src/scss/win/_media.scss

```diff
@@ -186,7 +186,7 @@
     }
 
     .win-media-price {
-        @include win-type-subcaption();
+        @include win-type-caption-alt();
```

Comment:
this comes from OneUI originally. do we want/need the notion of price in Winstrap?